tests/job.go

```diff
@@ -150,3 +158,15 @@ func NewHelloWorldJobHTTP(host string, port string, checkConnectivityCmdPrefixes
 func newHelloWorldJob(checkConnectivityCmd string) *batchv1.Job {
 	return NewJob("netcat", []string{"/bin/bash", "-c"}, []string{checkConnectivityCmd}, JobRetry, JobTTL, JobTimeout)
 }
+
+func NewHelloWorldJobHTTPWithDisabledIstioSidecar(host string, port string, checkConnectivityCmdPrefixes ...string) *batchv1.Job {
```

Comment:
Maybe pass the annotations as vararg to the original methods instead of introducing new ones.

Reply:
@AlonaKaplan Could you please elaborate?

Reply:
https://github.com/kubevirt/kubevirt/pull/5529 is removing `checkConnectivityCmdPrefixes`, so you can add `annotations ...string`

Reply:
Thank you. I ended up working around the issue, I realized that I only want to enable the sidecar in the upper-most BeforeEach, so I decided to label and de-label the namespace in this BeforeEach.